libs/design-system/src/lib/providers.ts

```diff
@@ -28,9 +28,9 @@ export function provideDesignSystemCommon(options?: DesignSystemOptions) {
     }),
     provideBrand(),
     provideButtons(),
-    provideChips(),
     provideCodeBlock(),
     provideIcons(),
+    provideRadioButton(),
```

Comment:
Remove. Should be provided in `provideButtons`. Add it there if not already provided